client/state/data-layer/wpcom/activity-log/rewind/to/index.js

```diff
@@ -32,7 +32,17 @@ const requestRestore = action =>
 			apiVersion: '1',
 			method: 'POST',
 			path: `/activity-log/${ action.siteId }/rewind/to/${ action.timestamp }`,
-			body: action.payload ? action.payload : {},
+		},
+		action
+	);
+
+const requestClone = action =>
+	http(
+		{
+			apiVersion: '1',
+			method: 'POST',
+			path: `/activity-log/${ action.siteId }/rewind/to/${ action.timestamp }`,
+			body: action.payload,
```

Comment:
cool! note that there's no reason we can't reuse this action creator here though…

```js
const requestRestore = action => requestRewind( action );
const requestClone = ( action, payload ) => requestRewind( action, payload );
```

diminishing returns probably but as you wish

Reply:
You mean something like 

```
const requestRewind = ( action, payload ) => 
	http(
		{
			apiVersion: '1',
			method: 'POST',
			path: `/activity-log/${ action.siteId }/rewind/to/${ action.timestamp }`,
			body: payload,
		},
		action
	);

const requestRestore = action => requestRewind( action );
const requestClone = action => requestRewind( action, action.payload );
```

or something else?